spras/rwr.py

```diff
@@ -0,0 +1,200 @@
+import warnings
+from pathlib import Path
+
+import pandas as pd
+
+from spras.containers import prepare_volume, run_container
+from spras.interactome import (
+    convert_undirected_to_directed,
+    reinsert_direction_col_directed,
+)
+from spras.prm import PRM
+from spras.util import add_rank_column
+
+__all__ = ['RWR']
+
+"""
+RWR will construct a directed graph from the provided input file
+- an edge is represented with a head and tail node, which represents the direction of the interation between two nodes
+- uses networkx Digraph() object
+
+Expected raw input format:
```

Comment:
What is edge flux for the input file requirements? And this is the network file, correct?